### DISCUSSION TITLE
Rows stopped working after upgrading

### DISCUSSION THREAD

**Discussion by carmenshu78** [onemen/TabMixPlus#358] [General]
Hi Everyone! After upgrading to 1.25 version, I can't see my tabs in rows like before. Is anyone experiencing the same?

**Comment by mdevil666** [comment]
Same problem here, even tried the dev build, broken too :(.

**Comment by Jankonyex** [comment]
Just upgraded and multi-row not working anymore.

**Comment by carmenshu78** [comment]
I downgraded to 1.24 version and the rows returned

**Comment by pyrates999** [comment]
go to: about:support

click on Clear startup cache... button.

**Reply by mdevil666** [reply]
Made the Downgrade to 1.24, rows working again, forgot to disable auto-update, so 1.25 got installed again, but rows  are working this time ... did not clear any caches

**Reply by evolved311** [reply]
> go to: about:support
> 
> click on Clear startup cache... button.

This worked, thanks!

**Reply by carmenshu78** [reply]
> go to: about:support
> 
> click on Clear startup cache... button.

It worked! Thanks!

**Reply by smihaila** [reply]
I'm on latest Firefox 132.0.1, and the yesterday's TabMixPlus 1.25 upgrade apparently broke the multi-row tabs feature. I had to downgrade back to the prior version aka 1.24, to make it work again. This situation was a gentle reminder for me to disable the auto-update option in Firefox' list of extensions.

Are you all saying that 1.25 is still good, under the condition that a "Clear Startup Cache" is performed?
How can us, the end-users know about such workaround apriori? It really panicked me, and I do not wish untested upgrades on my machine anymore!

**Reply by pyrates999** [reply]
yes, 1.25 is fine once you do what I said to do.

**Reply by GH0000001** [reply]
Same problem here (no multi row tabs with v1.25). 
Same solution: clear startup cache.

**Comment by onemen** [comment]
The fix already includes in the latest [dev-build](https://github.com/onemen/TabMixPlus/releases/tag/dev-build)
